pageserver/src/tenant/tasks.rs

```diff
@@ -363,7 +364,6 @@ async fn gc_loop(tenant: Arc<Tenant>, cancel: CancellationToken) {
                 first = false;
 
                 let delays = async {
-                    delay_by_lease_length(tenant.get_lsn_lease_length(), &cancel).await?;
```

Comment:
Aah I missed this initially, hmm so where do we configure the initial lsn length wait then...?